src/cljs/notifikator/events.cljs

```diff
@@ -1,9 +1,43 @@
 (ns notifikator.events
   (:require [re-frame.core :as re-frame]
             [notifikator.db :as db]
-            ))
+            [notifikator.util :refer [remove-by-id next-msg-id]]))
 
 (re-frame/reg-event-db
  ::initialize-db
  (fn [_ _]
    db/default-db))
+
+(defn handle-message
+  [db [_ title description] flavor]
+  (let [id (next-msg-id db)
+        msg {:id id
+             :title title
+             :description description
+             :class flavor}]
+    (update db :messages conj msg)))
+
+(re-frame/reg-event-db
+  ::spawn-message
+  (fn [db effect flavor]
```

Comment:
`reg-event-db` handler functions [only receive 2 arguments](https://github.com/Day8/re-frame/blob/master/src/re_frame/core.cljc#L75): `db` and `event-vector`, or in the more generic `reg-event-fx`, you receive `coeffects` and `event-vector`.